src/site/antora/modules/ROOT/pages/manual/simple-logger.adoc

```diff
@@ -0,0 +1,29 @@
+////
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+////
+
+= Simple Logger
+
+Even though Log4j Core is the reference implementation of xref:manual/api.adoc[Log4j API], Log4j API itself also provides a very minimalist implementation: link:../javadoc/log4j-api/org/apache/logging/log4j/simple/SimpleLogger.html[`SimpleLogger`].
+This is a convenience for environments where either a fully-fledged logging implementation is missing, or cannot be included for other reasons.
+`SimpleLogger` is the default Log4j API implementation if no other is available in the classpath.
+
+[#config]
+== Configuration
+
+`SimpleLogger` can be configured using the following system properties:
+
+include::partial$manual/systemproperties/properties-simple-logger.adoc[leveloffset=+1]
```

Comment:
There are two components of `SimpleLogger` configuration:

* the configuration of `SimpleLogger`,
* the configuration of `ThreadContextMap`.

Currently the latter is common with `log4j-core`, but there is no reason for it to be this way.
For example if `log4j2.simpleLogShowContextMap` is set to `false` (default), the `ThreadContextMap` should probably be a no-op.

Reply:
Fixed in 9a2815bb87b3bd0c8d7cde3ebefae095e948b049.